src/Bicep.Core/Text/TextSpan.cs

```diff
@@ -212,27 +210,29 @@ public static TextSpan Parse(string text)
             throw new FormatException($"The specified text span string '{text}' is not valid.");
         }
 
-        public static bool TryParse(string? text, out TextSpan span)
+        public static bool TryParse(string? value, out TextSpan span)
```

Comment:
Is this used in the product code anywhere? I could only find references to it in tests. If not, should we move it to a test library?